javascripts/content_scripts/privly.js

```diff
@@ -378,10 +372,11 @@ var privly = {
        };
 
     //Styling and display attributes
-     for(var key in attrs) 
-     {
-       iFrame.setAttribute(key, attrs[key]);
-     }
+	for(var key in attrs) {
```

Comment:
Ah yes. So a tab space of 2 spaces is what you guys work with? I'm using Brackets btw